runtime/vm/classsupport.c

```diff
@@ -301,26 +301,31 @@ internalFindClassString(J9VMThread* currentThread, j9object_t moduleName, j9obje
 		UDATA utf8Length = 0;
 		PORT_ACCESS_FROM_JAVAVM(vm);
 
-		if (NULL != moduleName) {
-			J9Module module = {0};
-			J9Module *modulePtr = &module;
-
-			modulePtr->moduleName = moduleName;
-			findResult = hashTableFind(classLoader->moduleHashTable, &modulePtr);
-			if (NULL != findResult) {
-				j9module = *findResult;
+		/* Make sure the name is legal */
+		if ((CLASSNAME_INVALID != allowedBitsForClassName)
```

Comment:
I believe this logic is wrong - should be (INVALID == bits) || (INVALID != functionCall)

Reply:
Fixed the logic error given `INVALID == bits` is accepted to avoid verification in `JVM_FindLoadedClass`